src/main/java/teammates/common/util/JsonUtils.java

```diff
@@ -23,8 +32,7 @@ private JsonUtils() {
      * Json file and also reformat the Json string in pretty-print format.
      */
     private static Gson getTeammatesGson() {
-        return new GsonBuilder().setDateFormat(DateFormat.FULL)
-                                .setDateFormat(Const.SystemParams.DEFAULT_DATE_TIME_FORMAT)
+        return new GsonBuilder().registerTypeAdapter(Date.class, new StandardDateAdapter())
```

Comment:
Their default adapter code can be found here: https://github.com/google/gson/blob/gson-parent-2.8.0/gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java